client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/site-preview-pane-header/style.scss

```diff
@@ -0,0 +1,61 @@
+		background-size: cover;
+		background-repeat: no-repeat;
+		height: 100px;
+	}
+
+	.site-preview__header-favicon {
+		position: relative;
+		top: -44px;
+		left: 48px;
+		width: 64px;
+		height: 64px;
+		margin-bottom: -64px;
+	}
+
+	.site-preview__header-content {
+		padding: 48px 48px 24px 48px;
+		display: flex;
+		align-items: center;
+
+		.site-preview__header-title-summary {
+			flex-grow: 1;
+			display: flex;
+			flex-direction: column;
+
+			.site-preview__header-title {
+				font-family: "SF Pro Text", sans-serif;
```

Comment:
It doesn't seem required

Reply:
[730c56b](https://github.com/Automattic/wp-calypso/pull/87760/commits/730c56ba50fd0a6a39657b8060de21c647ff595f)